application-services/custom/fledge-export/res/configuration.yaml

```diff
@@ -0,0 +1,22 @@
+Writable:
+  LogLevel: "INFO"
+  Telemetry:
+    Interval: "30s"
+
+Service:
+  Host: "localhost"
+  Port: 59780 # Adjust if running multiple examples at the same time to avoid duplicate port conflicts
+  StartupMsg: "LF Edge Fledge Export Service"
+
+MessageBus:
+  Disabled: false  # Set to true if not using edgex-messagebus Trigger below and Metrics are not enabled
+  Optional:
+    ClientId: "app-fledge-export"
+
+Trigger:
+  # Note that the MessageBus connection above is used by edgex-messagebus trigger which is the default set in common config
+  # Default value for SubscribeTopics is aslo set in common config
+  PublishTopic: "fledge-export"  # Base topic is prepened to this topic when using edgex-messagebus
```

Comment:
Example exports via HTTP, so no need to publish back to the EdgeX MessageBus
```suggestion
# Using default Trigger config from common config
```

Reply:
Done